models/articles-model.js

```diff
@@ -44,9 +44,28 @@ const fetchAllArticles = async (
   return db.query(query).then((response) => response.rows);
 };
 
+// const fetchArticleById = (id) => {
+//   return db
+//     .query("SELECT * FROM articles WHERE article_id = $1", [id])
+//     .then((response) => response.rows)
+//     .then((articles) => {
+//       if (articles.length === 0) {
+//         return Promise.reject({
+//           status: 404,
+//           msg: "Article not found",
+//         });
+//       } else {
+//         return articles[0];
+//       }
+//     });
+// };
+
```

Comment:
You shouldn't leave comments like this in when it comes to actually submitting the PR
If you want to keep notes of old chunks of code, consider making a file and adding it to .gitignore so it doesnt get pushed up to hithub